lms/djangoapps/teams/api.py

```diff
@@ -241,3 +244,27 @@ def can_user_create_team_in_topic(user, course_id, topic_id):
         (not is_instructor_managed_topic(topic_id)) or
         has_course_staff_privileges(user, course_id)
     )
+
+
+def get_team_for_user_and_course(user, course_id):
+    """
+    Returns the team that the given user is on in the course, or None
+
+    If course_id does not exist, a ValueError is raised
+    """
+    try:
+        course_key = CourseKey.from_string(course_id)
+        course_module = modulestore().get_course(course_key)
```

Comment:
Nit: move this (or the `CourseOverview` query) down after the `except` clause for clarity.